MapboxNavigation/StepsViewController.swift

```diff
@@ -0,0 +1,149 @@
+import UIKit
+import MapboxDirections
+import MapboxCoreNavigation
+
+class StepsViewController: UIViewController {
+    
+    weak var tableView: UITableView!
+    weak var closeButton: UIButton!
+    
+    let cellId = "StepTableViewCellId"
+    var steps: [RouteStep]!
+    let instructionFormatter = VisualInstructionFormatter()
+    
+    convenience init(steps: [RouteStep]) {
+        self.init()
+        self.steps = steps
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupViews()
+    }
+    
+    func setupViews() {
+        view.backgroundColor = #colorLiteral(red: 0.05205290616, green: 0.07546465523, blue: 0.1024953017, alpha: 1)
```

Comment:
Is this configurable?

Reply:
Good catch, will add a separate view for this.